onnxruntime/contrib_ops/cpu/sparse/sparse_attention_helper.h

```diff
@@ -36,7 +36,7 @@ Status CheckInputs(void* params,
   //   past_value           (batch_size, kv_num_heads, max_cache_sequence_length, head_size)
   //   block_row_indices    (num_layout, max_blocks + 1), where max_blocks = max_sequence_length / sparse_block_size
   //   block_col_indices    (num_layout, max_nnz)
-  //   seqlens_k_total      (batch_size) when do_rotary is True, optional otherwise
+  //   total_key_lengths    (batch_size)
```

Comment:
Since there is some renaming happening, I think there should be some note that makes it clear that the `seqlens_k` input from `GroupQueryAttention` corresponds to `total_key_lengths` here since `seqlens_k` is the named input in the GQA op spec.

Reply:
GroupQueryAttention has different spec for seqlens_k  like seqlens_k = total_key_lengths - 1.